Calling/ClientApp/src/core/sideEffects.ts

```diff
@@ -87,10 +88,22 @@ const subscribeToParticipant = (
   call: Call,
   dispatch: Dispatch
 ): void => {
+  let remoteStreamSelector = RemoteStreamSelector.getInstance(1, dispatch);
```

Comment:
hoping we wouldnt be using magic numbers here and instead have a constant defined somewhere